src/password.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PASSWORD_H
+#define PASSWORD_H
+
+#include <cstring>
+
+const std::string read_password(const char* file);
+
+const std::string eval_password(const char* cmd);
+
```

Comment:
I'd rather have these two functions take `const std::string&`. Newsbeuter is a C++ project, after all :)